src/main/java/castalia/Dist.java

```diff
@@ -0,0 +1,381 @@
+            result = Math.exp(-xsq*xsq*half) * Math.exp(-del*half)*result;
+            ccum = one - result;
+            if (x > zero) { temp = result; result = ccum; ccum = temp; }
+        }
+        if (result < min) { result = 0.0; }
+        if (ccum < min) { ccum = 0.0; }
+        return result;
+    }
+
+    /** integer floor */
+    public static double fint(double x)
+    { return (x >= 0.0) ? Math.floor(x) : -Math.floor(-x); }
+
+
+    // From R source
+    /** standard normal quantile function */
+    public static double qnorms(double p)
+    { double q, r, val;
+        double a0 = 2.50662823884;
+        double a1 = -18.61500062529;
+        double a2 = 41.39119773534;
+        double a3 = -25.44106049637;
+        double b1 = -8.47351093090;
+        double b2 = 23.08336743743;
+        double b3 = -21.06224101826;
+        double b4 = 3.13082909833;
+        double c0 = -2.78718931138;
+        double c1 = -2.29796479134;
+        double c2 = 4.85014127135;
+        double c3 = 2.32121276858;
+        double d1 = 3.54388924762;
+        double d2 = 1.63706781897;
+        double zero = 0.0;
+        double half = 0.5;
+        double one = 1.0;
+        double split = 0.42;
+
+        if (p <= 0.0 || p >= 1.0) return Double.NaN;
+        q = p - half;
+        if (Math.abs(q) <= split)
+        { /* 0.08 < p < 0.92 */
+            r = q * q;
+            val = q * (((a3 * r + a2) * r + a1) * r + a0)
+                    / ((((b4 * r + b3) * r + b2) * r + b1) * r + one);
+        }
+        else
+        { /* p < 0.08 or p > 0.92, set r = min(p,1-p) */
+            r = p;
+            if (q > zero) r = one - p;
+            r = Math.sqrt(-Math.log(r));
+            val = (((c3 * r + c2) * r + c1) * r + c0)
+                    / ((d2 * r + d1) * r + one);
+            if (q < zero) val = -val;
+        }
+        val = val - (pnorms(val) - p) / dnorms(val);
+        return val;
+    }
+
+
+
+
+    public static int signgam = 0;
+  /* the sign of gamma(x), set in lgamma(.), used by gamma(.) */
+
+    //static double Gam_pos(double);
+    //static double lGam_neg(double);
+    //static double asform(double); /* ASymptotic FORM */
+
+    /** log gamma function */
+    public static double lgamma(double x)
+    {  signgam = 1;
+        if (x <= 0.0) return (lGam_neg(x));
+        if (x >  8.0) return (asform(x));
+        return (Math.log(Gam_pos(x)));
+    }
+
+    /* Coefficients  from Cheney and Hart (??--REF---??) */
+   /* Asymptotic form: (not quite, why are the coeff. slightly changed ? */
+    //#define M 6
+    public static double asform(double x)
+    {
+    /* Equation 6.1.41 Abramowitz and Stegun -- extended Stirling */
+    /* See also ACM algorithm 291 */
+
+    /*	double log();*/
+        int M=6;
+        double p1[] = { 0.83333333333333101837e-1, -.277777777735865004e-2,
+                0.793650576493454e-3, -.5951896861197e-3,
+                0.83645878922e-3, -.1633436431e-2, };
+        double nfac, xsq;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://www.codacy.com/app/scala-academy/Castalia/file/1968742700/issues/source?bid=2998213&fileBranchId=3112713#l203)